proto/examples/calculator/v1/calculator.proto

```
@@ -0,0 +1,98 @@
+syntax = "proto3";
+
+package calculator.v1;
```

Buf (1.55.1)

Files with package "calculator.v1" must be within a directory "calculator/v1" relative to root but were in directory "proto/examples/calculator/v1".

(PACKAGE_DIRECTORY_MATCH)